app/resources/en/translation/deceased/dob.json

```diff
@@ -7,39 +7,47 @@
   "errors": {
     "dob_day": {
       "required": {
-        "summary": "Add the date of birth of the person who has died ",
-        "message": "Enter date of birth"
+        "summary": "You haven&rsquo;t entered the day that they were born",
```

Comment:
I've talked to @LucaDelBuono about the use of escaped chars here
We should investigate whether this is strictly necessary

Reply:
It's something that we've done throughout the app...it was a decision by UX a long time ago to have proper 66 and 99 quotes and keep them consistent between pages.

We should continue to do this for new/updated content and if we want to change it then we should do so across the app.

Reply:
It was decided a long time ago to use &rsquo;t rather than a quote as it aesthetically looks better.  This approach is taken throughout our codebase and we have not had any feedback to say we shouldn't use &rsquo;
Thus, I'd suggest we should leave this as-is and address it if it is reported as an issue by GDS or our users